src/index.html

```diff
@@ -20,8 +20,504 @@
       href="styles/index.scss"
     />
   </head>
-
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="header__link"
+      >
+        <img
+          class="logo"
+          src="images/logo.png"
+          alt="logo"
+        />
+      </a>
+      <nav class="nav header__nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              class="nav__link is-active"
+              href="#"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+              Photo
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main
+      class="container"
+      data-qa="card"
+    >
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <div class="card__photo card__photo--margin">
+          <img
+            class="card__image"
+            src="/src/images/imac.jpeg"
+            alt="imac"
+          />
+        </div>
+
+        <div class="card__title card__title--margin">
+          <p>APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</p>
+        </div>
+
+        <div class="card__subtitle card__subtitle--margin">
+          <p>Product code: 195434</p>
+        </div>
+
+        <div class="card__feedback feedback feedback--margin">
```

Comment:
The same nesting error is happening here! Read the comment above - also check the exception that I wrote below